### PR TITLE
fix link

### LINKED ISSUE #15534: Run reference contains a bad link
Found a dead link in logging run reference: docs.docker.com/reference/run/…

### PR DESCRIPTION
**Pull request by MHBauer**
resolves  #15534

### CODE REVIEW

File: docs/reference/run.md [hunk 1]
```diff
@@ -908,7 +908,7 @@ container's logging driver. The following options are supported:
 
 	The `docker logs`command is available only for the `json-file` logging
 driver.  For detailed information on working with logging drivers, see
-[Configure a logging driver](reference/logging/overview.md).
+[Configure a logging driver](/reference/logging/overview).
```
Comment (thaJeztah) [inline docs/reference/run.md]: nit: I think this needs a trailing slash (`/reference/logging/overview/`) to prevent an extra redirect

Comment (MHBauer) [inline docs/reference/run.md]: I can change this one.

There are a lot of links without a trailing slash out there. Would it help to change those? I do not have any insight into how the server works. What can I do to see what is happening behind the scenes? Is there debug/verbosity flag for the hugo server in `make docs`?

**Comment by moxiegirl** [comment]
LGTM

**Comment by thaJeztah** [comment]
Oh! Missed the ping.

LGTM

thanks @MHBauer !

**Comment by MHBauer** [comment]
should close #15534

**Comment by thaJeztah** [comment]
Thanks for the reminder! apparently GitHub doesn't like "resolves" now? I know "fixes" works..

**Pull request merged**